### ISSUE TITLE
Unable to add more than 353 accounts in Edge Extensions for Authenticator

### ISSUE THREAD

**Issue by dennyjohny** [Authenticator-Extension/Authenticator#1101]
### Describe the issue

Hi Team,

I have 3 issues:

1. I am using Edge Extensions for Authenticator in my edge browser. Currently I have 353 accounts added into this and I am unable to add 354th account. Is it a limitation ?
2. In chrome I can see the version 6.3.5 which updated on May 10, 2023. But in edge I can only see 6.3.3 which is last updated on Updated October 22, 2021. Why latest version is not available in edge ?
3. I have created a backup from Edge with 353 accounts and tried to restore in chrome extension but it is not working. When I removed half of the accounts it worked. Why is this behavior ?

Requesting your support.

Regards,
Denny Johny

### Browser

Edge

### Browser Version

Version 116.0.1938.62 (Official build) (64-bit)

### Extension Version

6.3.3

**Comment by Sneezry** [comment]
There is 100k limit for sync storage, and I guess you have ran out the quota

**Comment by dennyjohny** [comment]
Thank you @Sneezry. Is there any option to increase it. Also how I can cross check this. Where I can see this limit ?

**Comment by Clickomatic** [comment]
We have over 3,000 users who we'd like to use this with from end Jan 2024, am glad I discovered this limit now. 
Echoing your question @dennyjohny also want to know how to increase sync storage limit to I guess  over 1.5 MB.

**Comment by dennyjohny** [comment]
Can someone help to answer this ?

Unable to add more than 353 accounts in Edge and chrome Extensions for Authenticator

**Comment by TESTER-sec** [comment]
People are crazy for continuing to use Authenticator as it is very obviously not properly supported.

So many reported problems are never addressed, let alone fixed. Then getting support is next to impossible.

I cannot believe that Authy recommended this software as a replacement to their Desktop app which they are killing-off in August 2024.

**Comment by mymindstorm** [comment]
@dennyjohny @Clickomatic we don't test with or recommend having that many accounts, so be aware that you might run into other issues. If you turn off "Browser Sync", the data will not sync, but should increase the storage limit. We cannot control the limit of the sync storage due to limits from Google and Mozilla.

**Comment by TESTER-sec** [comment]
> @dennyjohny @Clickomatic we don't test with or recommend having that many accounts, so be aware that you might run into other issues. If you turn off "Browser Sync", the data will not sync, but should increase the storage limit. We cannot control the limit of the sync storage due to limits from Google and Mozilla.

Please explain to us how we are not supposed to have that many accounts whenever every last one of those accounts is required?

Not everybody has 100 or less accounts.

Also, are you going to get around to answering or addressing all the issues being reported? Most have no reply. That does not inspire confidence. It really is obvious that this software is not properly supported.

**Comment by mymindstorm** [comment]
> Please explain to us how we are not supposed to have that many accounts whenever every last one of those accounts is required?

It's not that you aren't supposed to have that many accounts, more of that we don't currently test with that many accounts and there may be some issues that we are not aware of.

> Also, are you going to get around to answering or addressing all the issues being reported? Most have no reply. That does not inspire confidence. It really is obvious that this software is not properly supported.

We are a volunteer open source project and do not have a lot of time to work on the extension, nor do we provide a guarantee of support. You are free to start contributing if you feel that the extension could be improved somewhere or resolve an issue.

**Comment by TESTER-sec** [comment]
If it is a volunteer project software with next to no support, then that fact needs to be posted in large font on the Authenticator webpages.

People are relying upon this software for TOPT logins and when it doesn't work correctly, they can be locked out of their accounts.

Authenticators are in the same class as password managers. They are critical and require to be fully supported on a full-time, fully staffed basis.

I would bet that 99% of people don't know what they are getting themselves into when they choose Authenticator. They only find out that it isn't properly supported until problems begin - which is a real problem because Authenticator has a lot of problems, and they invariably do not get fixed.

**Comment by dennyjohny** [comment]
Hi @mymindstorm 

Thank you for the response. I would like to bring to your attention that we are not currently utilizing Browser Sync. Therefore, I hope any associated limits do not apply to our situation.

I fully comprehend the outlined testing and support scope. However, I kindly request your consideration in investigating the specific issue we are encountering.

The reason for this request is that if there is no possibility of testing and fix from the support team regarding this matter, it could have a significant impact on our operations. Regrettably, we have been unable to identify an alternative solution at this time.

I sincerely appreciate your understanding of our situation, and I am hopeful that we can explore a feasible resolution together. Your support in this matter would be greatly appreciated.

**Comment by mymindstorm** [comment]
Could you follow [these steps](https://authenticator.cc/docs/en/debug-info), try to import the large amount of accounts, and post a screenshot of the logs here? 

> Thank you for the response. I would like to bring to your attention that we are not currently utilizing Browser Sync. 

To clarify, does this mean that you have disabled sync in the browser or that you have set the "browser sync" from the below screenshot to off?

![image](https://github.com/Authenticator-Extension/Authenticator/assets/27789806/e199aac5-1689-445f-996a-dbf93692de56)

**Comment by Sneezry** [comment]
@mymindstorm Perhaps we could gently recommend to customers the option of switching to local area storage and seeking unlimitedStorage permission if they find themselves managing an excessive number of accounts. In the long run, we could consider implementing a feature that allows customers to choose which accounts to store in the synchronization area, leaving the rest in their local area storage.

**Comment by mymindstorm** [comment]
@Sneezry Agree, just want to make sure that this is not due to the page using too much memory or some other edge case.

**Comment by dennyjohny** [comment]
Thank you @mymindstorm & @Sneezry. I will check thease options and will test. I will update after the test.

**Comment by dennyjohny** [comment]
Thank you @mymindstorm @Sneezry - I Stopped Browser Sync. It seems worked. I am able to proceed and so far no issues.
